documentation/docmaker.json

```diff
@@ -1639,6 +1639,23 @@
                 }
               ],
               "md": "collection/layoutPositions"
+            },
+
+            {
+              "name": "nodes.layoutDimensions",
+              "descr": "Returns the node width and height",
```

Comment:
Though it's important to be concise in descriptions, it's also important to be clear enough to give users a sense of how one function differs from another.  I think users might confuse this function with ones like `width()`, `height()`, and `boundingBox()`.  Maybe it's worth mentioning that it's for layout positioning (e.g. overlap detection) in the top description?

By the way, you can run `gulp docs` to see your spec changes reflected in the docs page.